Makefile

```diff
@@ -0,0 +1,13 @@
+all: 05_final_report.pdf
+
+original_data:
```

Comment:
@jhofman Our data doesn't seem to need any cleaning. Can we get rid of script 02? or do you prefer we separate out the data loading in case we ever want to do cleaning.

Reply:
@calvinytong: sure, restructure as needed.

Reply:
Also, you mentioned that there might be some temporal data sitting around somewhere, but we are having trouble finding it. Any leads on that?

Reply:
I don't believe there's anything publicly available with timestamps, unfortunately. Too bad!